### PR TITLE
added info that user should download meta mask

### PR DESCRIPTION
**Pull request by nexus-codingg**
added information for adding metamask extension for web3 login flow

### CODE REVIEW

File: docs/users/web3.mdx [hunk 1]
```diff
@@ -11,6 +11,8 @@ Learn how to use Clerk to quickly and easily add secure authentication and user
 
 You need to create a Clerk application in your [Clerk Dashboard](https://dashboard.clerk.com/) and configure the application to use **Sign in with Metamask**. For more information on how to set up a Clerk application, check out the [setup guide](/docs/quickstarts/setup-clerk). To enable Metamask authentication, check out [this guide](/docs/authentication/configuration/sign-in-options#social-o-auth-web3).
 
+Please ensure that you have downloaded [Metamask](https://metamask.io/download/) on your machine to ensure a seamless login flow.
```
Comment (royanger) [inline docs/users/web3.mdx:14]:
```suggestion
Please ensure that you have downloaded the [Metamask plugin](https://metamask.io/download/) and installed it on your browser to ensure a seamless login flow.
```

**Review by royanger — CHANGES_REQUESTED** [review]
Please update the info, other wise looks great!

File: docs/users/web3.mdx (hunk 1 above)
Comment (nexus-codingg) [reply docs/users/web3.mdx]: added changes

**Review by royanger — APPROVED** [review]
Looks good

**Pull request merged**